cmake/Modules/Findrav1e.cmake

```diff
@@ -64,5 +64,10 @@ if(RAV1E_LIBRARY)
         endif()
         set_target_properties(rav1e::rav1e PROPERTIES IMPORTED_LOCATION "${RAV1E_LIBRARY}" IMPORTED_SONAME rav1e)
         target_include_directories(rav1e::rav1e INTERFACE ${RAV1E_INCLUDE_DIR})
+        if(WIN32)
+            target_link_libraries(rav1e::rav1e INTERFACE ntdll.lib userenv.lib ws2_32.lib bcrypt.lib)
+        elseif(UNIX AND NOT APPLE)
+            target_link_libraries(rav1e::rav1e INTERFACE ${CMAKE_DL_LIBS}) # for backtrace
+        endif()
```

Comment:
For future maintainers, it would be good to cite the source of this information. This is not obvious.

I am also wondering why we didn't need this before. Does this fix a bug that affects our clients but doesn't affect our own build?

Reply:
These instructions were simply taken from the main CMake. We need to unify that.
We did not see it because this happens when rav1e is installed. Our CI only has local rav1e. I'll try adding one with installed packages on Window.s